### DISCUSSION TITLE
Cant find and or add a symbol

### DISCUSSION THREAD

**Discussion by Herrcookiekiller** [fontforge/fontforge#5269] [Q&A]
How do i find the € symbol in the "ISO 10646-1 (Full)" symbol set?
Also i cant find any documentation on how to add missing ascii symbols by yourself in the wiki, or am i missing something?

**Comment by Bry10022** [comment]
You can scroll to a point that is close to the codepoint you want, then press Page Up or Page Down until you can see the Euro Symbol. An easier way is to switch the encoding to one that contains the Euro symbol, like Windows Latin aka ANSI.

**Reply by Herrcookiekiller** [reply]
Ahh yes thank you very much, i googled the unicode for € and scrolled down in the chart until i found it. Also i assume that theres every Ascii symbol possible in that ISO symbol set and you just only add the ones you need and FontForge leaves out the ones that arent added by the user.
